src/txproto_main.c

```diff
@@ -2217,8 +2217,8 @@ int main(int argc, char *argv[])
         if ((err = lfn_loadfile(ctx, script_name)))
             goto end;
     } else {
-        err = luaL_loadbufferx(ctx->lua, scripts_default_lua_bin,
-                               scripts_default_lua_bin_len, "built-in script", "b");
+        err = luaL_loadbufferx(ctx->lua, default_lua_bin, default_lua_bin_len,
+                               "built-in script", "b");
```

Comment:
Tried to keep the indentation but might be a good idea to have something like a `.clang-format` file or a set of options to pass to `indent(1)`